src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -41,14 +42,9 @@
 public class SDKClient {
     private OpenSearchClient javaClient;
     private RestClient restClient = null;
```

Comment:
Why is this one initialized null but the high level client isn't?  I presume null is the default in either case, but I'd like to be more intentional here on how these are initialized and/or null-checked.